src/renderers/webgl/WebGLProgram.js

```diff
@@ -377,6 +377,93 @@ function generateEnvMapBlendingDefine( parameters ) {
 
 }
 
+function generateStandardDefines( renderer, parameters, extensions, material ) {
```

Comment:
I guess I would rename the method to `generateDefines()`. The term `Standard` is already related to `MeshStandardMaterial`.